insights/client/__init__.py

```diff
@@ -50,6 +50,14 @@ def __init__(self, config=None, from_phase=True, **kwargs):
         if from_phase:
             _init_client_config_dirs()
             self.set_up_logging()
+            logger.debug(
+                "phase={phase}, version={version};{path}, arguments={arguments}".format(
+                    phase=os.getenv("INSIGHTS_PHASE", "unknown"),
+                    version=package_info.get("VERSION", "unknown"),
+                    path=client.__file__,
+                    arguments=" ".join(sys.argv[1:]),
```

Comment:
Those are not settable via CLI, only through the configuration file. Should we try to filter them anyways, in case someone enters it there by mistake?